README.adoc

```diff
@@ -2,6 +2,8 @@
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
 https://travis-ci.com/AY1920S1-CS2103T-F14-4/main[image:https://travis-ci.com/AY1920S1-CS2103T-F14-4/main.svg?branch=master[Build Status]]
+https://coveralls.io/github/AY1920S1-CS2103T-F14-4?branch=master[image:https://coveralls.io/repos/github/AY1920S1-CS2103T-F14-4/badge.svg?branch=master[Coverage Status]]
```

Comment:
Didn't specify the `main` repo. See the netlify preview, this is not linked correctly